CompetitionCode/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -68,8 +68,8 @@
   public final InstantCommand pistonOffCommand = new InstantCommand(() -> INTAKE.pistonOff(), INTAKE);
 
   // for storage trigger
-  public boolean shouldStorageIntake() {
-    return STORAGE.isSwitchPressed(0) && !STORAGE.isOverridden();
+  public boolean storageShouldIntake() {
```

Comment:
this should be 'shouldStorageIntake()'. It's more straight forward when viewing it for the first time.